### DISCUSSION TITLE
Save full html articles locally

### DISCUSSION THREAD

**Discussion by nanotechz9l** [ViennaRSS/vienna-rss#374] [Ideas]
It would be a nice feature to be able to save full HTML articles by default. As it stands now, we can only view our feeds online. It would be huge to be able to see our cached articles using this feature. This will benefit users on long flights, train rides, or for those that simply don't want to connect to untrusted public wifi.

**Comment by berliner** [comment]
Definitely interested in this feature too. I'm often enough on slow wifi connections and I'm regularly surprised that this not the case already.
